Source/Core/VideoSynchronizer.js

```diff
@@ -172,7 +172,12 @@ define([
             return;
         }
 
-        element.playbackRate = clock.multiplier;
+        try {
+            element.playbackRate = clock.multiplier;
```

Comment:
It's bad to have try/catch blocks in a hot area of the code, the optimizer can't do a good job and possibly throwing an exception every frame is really slow.  Is this only for negative rates?  Can we check once up front for negative values support and then just use a simple boolean here?

Reply:
Yeah, good point. It's annoying but the exception can occur for large positive values too. Different browsers have different limits*. If we try to assign `playbackRate` out of the acceptable range without handling the exception, it seems to bubble up and cause the Viewer widget to freeze like in #6765.

What if we mark the value as invalid if an exception is thrown and then refuse to try again until it's changed? We can reset the actual `playbackRate` to 1.0 and let the clock adjustment take over. If in the `CesiumViewer` widget we ignore this exception specifically we can be a little less picky about what we assign to `playbackRate`.

\* (Chrome and Firefox only accept rates from 0.0625 to 16.0 according to [this SO answer](https://stackoverflow.com/a/32320020). I don't know about Safari or Edge, [but apparently playbackRate doesn't work at all on iOS](https://developer.apple.com/documentation/webkitjs/htmlmediaelement/1629746-playbackrate), we should probably keep that in mind.)

Reply:
Caching the last value and only setting on change sounds like a good idea.  I would create a private helper function, `trySetPlaybackRate` that does it, so that the try/catch block is only inside that local helper and is a no-op if the old value and new value are the same.  You need to keep a separate `lastPlaybackRate` variable since there's no garuantee `element.playbackRate` will match the desired balue.